### ISSUE TITLE
Typo in Chapter 7

### ISSUE THREAD

**Issue by Ashuradhipathi** [mrdbourke/pytorch-deep-learning#1089]
In the 8th sub part of Chapter 7, In the caption for the picture, It was stated " you can see that the EffNetB0 model trained for 10 epochs and with 20% of the data achieves the lowest loss.". The actual model was "EffNetB0". By the way, Thank you. It has been a great resource for learning Pytorch. I am very grateful.

**Comment by pritesh2000** [comment]
Hey @Ashuradhipathi,
Can you explain what is typo here?

**Comment by Ashuradhipathi** [comment]
Yeah sure the model is B12 but was written B10 in the caption

**Comment by Ashuradhipathi** [comment]
Sorry, It should be EffNetB2 but was written EffnetB0 in the caption.

**Comment by pritesh2000** [comment]
> Sorry, It should be EffNetB2 but was written EffnetB0 in the caption.

In the image caption https://www.learnpytorch.io/07_pytorch_experiment_tracking/#8-view-experiments-in-tensorboard
You can see in the image the highlighted experiment in the image, the mentioned experiment is covered with dotted green line.

test loss for EffNetB2, 10 epochs and 20% data is 0.3909
test loss for EffNetB0, 10 epochs and 20% data is 0.2771

I don't think there is a change needs to be made, If I misunderstand something please mention it.

**Comment by Ashuradhipathi** [comment]
I am sorry, I mistook loss for accuracy, Thank you for clarifying. After reading " This sticks with the overall trend of the experiments that: more data, larger model and longer training time is generally better." I thought shouldnt B2 have less loss and saw accuracy graph and mistook it for loss graph

**Comment by pritesh2000** [comment]
> I am sorry, I mistook loss for accuracy, Thank you for clarifying. After reading " This sticks with the overall trend of the experiments that: more data, larger model and longer training time is generally better." I thought shouldnt B2 have less loss and saw accuracy graph and mistook it for loss graph

Here, If we have more data and to feed B2 model then it would definitely improve. But we are using small subset of data that is the reason B0 is having higher accuracy than B2.
